RecoEgamma/EgammaElectronProducers/plugins/GsfElectronProducer.cc

```diff
@@ -56,22 +56,22 @@ namespace {
       const auto& dnn_ele_pfid = hoc->iElectronDNNEstimator->evaluate(electrons, tfSessions);
       int jele = 0;
       for (auto& el : electrons) {
-        const auto& values = dnn_ele_pfid[jele];
+        const auto& [iModel, values] = dnn_ele_pfid[jele];
         // get the previous values
         auto& mvaOutput = mva_outputs[jele];
 
-        if (abs(el.superCluster()->eta()) <= extetaboundary) {
-          mvaOutput.dnn_e_sigIsolated = values[0];
-          mvaOutput.dnn_e_sigNonIsolated = values[1];
-          mvaOutput.dnn_e_bkgNonIsolated = values[2];
-          mvaOutput.dnn_e_bkgTau = values[3];
-          mvaOutput.dnn_e_bkgPhoton = values[4];
-        } else {
-          mvaOutput.dnn_e_sigIsolated = values[0];
+        if (iModel <= 3) {  // models 0,1,2,3 have 5 outpus in this version
```

Comment:
minor typo in comment
```suggestion
        if (iModel <= 3) {  // models 0,1,2,3 have 5 outputs in this version
```